topics/variant-analysis/tutorials/microbial-variants/tutorial.md

```diff
@@ -52,43 +52,51 @@ This data is available at Zenodo using the following [link](https://doi.org/10.5
 > 1.  Import all of the following files into a new history:
 >     - [mutant_R1.fastq](https://zenodo.org/record/582600/files/mutant_R1.fastq)
 >     - [mutant_R2.fastq](https://zenodo.org/record/582600/files/mutant_R2.fastq)
->     - [wiltype.fna](https://zenodo.org/record/582600/files/wildtype.fna)  
->     - [wiltype.gbk](https://zenodo.org/record/582600/files/wildtype.gbk)
->     - [wiltype.gff](https://zenodo.org/record/582600/files/wildtype.gff)
->
->    > ### {% icon tip %} Tip: Importing data via links
->    >
->    > * Copy the link location
->    > * Open the Galaxy Upload Manager
->    > * Select **Paste/Fetch Data**
->    > * Paste the link into the text field
->    > * Press **Start**
->    {: .tip}
+>     - [wildtype.fna](https://zenodo.org/record/582600/files/wildtype.fna)  
+>     - [wildtype.gbk](https://zenodo.org/record/582600/files/wildtype.gbk)
+>     - [wildtype.gff](https://zenodo.org/record/582600/files/wildtype.gff)
+> 
+>     ```
+>     https://zenodo.org/record/582600/files/mutant_R1.fastq
+>     https://zenodo.org/record/582600/files/mutant_R2.fastq
+>     https://zenodo.org/record/582600/files/wildtype.fna
+>     https://zenodo.org/record/582600/files/wildtype.gbk
+>     https://zenodo.org/record/582600/files/wildtype.gff
+>     ```  
+>
+>     > ### {% icon tip %} Tip: Importing data via links
+>     >
+>     > * Copy the link location
+>     > * Open the Galaxy Upload Manager
+>     > * Select **Paste/Fetch Data**
+>     > * Paste the link into the text field
+>     > * Press **Start**
+>     {: .tip}
 >
 {: .hands_on}
 
-Your history pane should look like this after you have uploaded all of the data:
+# Find variants with Snippy
 
-![micro-start-history.png](../../images/micro-start-history.png)
```

Comment:
Is there a reason we get rid of this picture? We normally have them in place so the user can see that they are on the right track in the beginning for a bit of confidence etc.

Reply:
I agree with @Slugger70 - for users who are new to uploading to Galaxy it useful to show the starting point.

Reply:
The rational behind removing images is that they are hard to update in general. Also please take into account that the "Galaxy introduction" is a pre-requirement to this tutorial, so I do not think it is so much useful at this point.